Assets/Tests/PlayModeTests/ViewsTests.cs

```diff
@@ -66,21 +66,18 @@ public async void ViewNameLimit()
 
             await views.RecordOpenViewAsync("open_view");
             await views.RecordCloseViewAsync("close_view");
-            TestUtility.ValidateRQEQSize(cly, 2, 2);
+            TestUtility.ValidateRQEQSize(cly, 2, 1);
 
             CountlyEventModel model = cly.Events._eventRepo.Dequeue();
             TestUtility.ViewEventValidator(model, 1, null, null, TestUtility.BaseViewTestSegmentation("open_", true, true), null, null, null, null, TestUtility.TestTimeMetrics());
-
-            model = cly.Events._eventRepo.Dequeue();
-            TestUtility.ViewEventValidator(model, 1, null, null, TestUtility.BaseViewTestSegmentation("close", false, false), null, null, null, null, TestUtility.TestTimeMetrics());
             TestUtility.ValidateRQEQSize(cly, 2, 0);
         }
 
         // 'RecordCloseViewAsync' method in ViewCountlyService.
-        // We close a view. Verify that the event is correctly recorded in the Views repository
-        // If a valid view name is provided, it should be recorded and EventModel should be validated
+        // We try to close a view that's not existing
+        // If view does not exist nothing should be recorded, nothing should crash
         [Test]
-        public async void RecordCloseViewAsync()
+        public async void RecordCloseViewAsync_NoOpenView()
```

Comment:
maybe you could leave this test case and make it a failing test case
and create a new valid test case

Reply:
in my opinion it was testing something that shouldn't already be there, so leaving it didn't make much sense to me, if others agree too yes surely we can keep that one and create another case